core/src/dispatch/mod.rs

```diff
@@ -413,6 +475,11 @@ impl NetworkDispatcher {
                 );
                 self.queue_manager.drop_queue(&addr);
                 self.connections.remove(&addr);
+                self.network_status_port
+                    .trigger(NetworkStatus::ConnectionDropped(SystemPath::with_socket(
+                        Transport::TCP,
+                        addr,
```

Comment:
Is this correctly canonicalised if we have multiple channels with the same system open?

Reply:
At the moment we don't support multiple channels. I will make sure that this is correctly canonicalised when I implement that feature.